### PR TITLE
ifstat: Add package

### PR DESCRIPTION
**Pull request by nikil**
Maintainer: me
Compile tested: ar71xx, x86_64
Run tested: x86_64

Description: InterFace STATistics Monitoring

Signed-off-by: Nikil Mehta nikil.mehta@gmail.com

### CODE REVIEW

File: net/ifstat/Makefile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
```
Comment (diizzyy) [inline net/ifstat/Makefile]: Please don't use OpenWrt.org copyright unless you are under a contract.
https://www.lede-project.org/faq.html#who_holds_the_copyright_of_lede_code

File: net/ifstat/Makefile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ifstat
+PKG_VERSION:=1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://gael.roualland.free.fr/ifstat/
+PKG_MD5SUM:=b655642c33a626cfe976792fbcd9b6e1
```
Comment (diizzyy) [inline net/ifstat/Makefile]: Please use SHA256 instead of MD5 checksums

**Comment by nikil** [comment]
Done - thanks for the comments.

**Comment by diizzyy** [comment]
@hnyman @thess 
I can see a point in having this with netsnmp but it doesn't make much sense without this extension. Ok to merge?

**Comment by thess** [comment]
Sure - thanks

**Pull request merged**